tools/site_and_regional/run_neon.py

```diff
@@ -389,19 +364,25 @@ def run_case(self, base_case_root, run_type, run_length, overwrite=False):
                 case.set_value("CLM_ACCELERATED_SPINUP","on")
                 case.set_value("RUN_REFDATE", "0018-01-01")
                 case.set_value("RUN_STARTDATE", "0018-01-01")                
+                case.set_value("JOB_WALLCLOCK_TIME", "12:00:00", subgroup="case.run")
             else:
                 case.set_value("CLM_FORCE_COLDSTART","off")
                 case.set_value("CLM_ACCELERATED_SPINUP","off")
                 case.set_value("RUN_TYPE", "hybrid")
                 
             if run_type == "postad":
                 self.set_ref_case(case)
+                case.set_value("REST_OPTION","nmonths")
+                case.set_value("REST_N", 12)
                 
             if run_type == "transient":
-                self.set_ref_case(case)
+                if self.finidat:
+                    case.set_value("RUN_TYPE","startup")
+                else:
+                    self.set_ref_case(case)
                 case.set_value("STOP_OPTION","nmonths")
                 case.set_value("STOP_N", self.diff_month())
-                case.set_value("REST_N", "12")
+                case.set_value("REST_OPTION", "end")
```

Comment:
does end need to be defined, or does this just take us through the last datm file that's found?

Reply:
end is the end of the run, it's defined in terms of STOP_N and STOP_OPTION.
This is set for transient to the end of the last datm file found.